2024-02-27 19:42:22,387 INFO    StreamThr :19880 [internal.py:wandb_internal():86] W&B internal server running at pid: 19880, started at: 2024-02-27 19:42:22.386892
2024-02-27 19:42:22,388 DEBUG   HandlerThread:19880 [handler.py:handle_request():146] handle_request: status
2024-02-27 19:42:22,397 INFO    WriterThread:19880 [datastore.py:open_for_write():87] open: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240227_194222-73trvyvh/run-73trvyvh.wandb
2024-02-27 19:42:22,398 DEBUG   SenderThread:19880 [sender.py:send():382] send: header
2024-02-27 19:42:22,401 DEBUG   SenderThread:19880 [sender.py:send():382] send: run
2024-02-27 19:42:23,014 INFO    SenderThread:19880 [dir_watcher.py:__init__():211] watching files in: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240227_194222-73trvyvh/files
2024-02-27 19:42:23,014 INFO    SenderThread:19880 [sender.py:_start_run_threads():1136] run started: 73trvyvh with start time 1709043142.386181
2024-02-27 19:42:23,021 DEBUG   HandlerThread:19880 [handler.py:handle_request():146] handle_request: check_version
2024-02-27 19:42:23,021 DEBUG   SenderThread:19880 [sender.py:send_request():409] send_request: check_version
2024-02-27 19:42:23,122 DEBUG   HandlerThread:19880 [handler.py:handle_request():146] handle_request: run_start
2024-02-27 19:42:23,126 DEBUG   HandlerThread:19880 [system_info.py:__init__():27] System info init
2024-02-27 19:42:23,126 DEBUG   HandlerThread:19880 [system_info.py:__init__():42] System info init done
2024-02-27 19:42:23,126 INFO    HandlerThread:19880 [system_monitor.py:start():194] Starting system monitor
2024-02-27 19:42:23,126 INFO    SystemMonitor:19880 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-02-27 19:42:23,127 INFO    HandlerThread:19880 [system_monitor.py:probe():214] Collecting system info
2024-02-27 19:42:23,127 INFO    SystemMonitor:19880 [interfaces.py:start():190] Started cpu monitoring
2024-02-27 19:42:23,128 INFO    SystemMonitor:19880 [interfaces.py:start():190] Started disk monitoring
2024-02-27 19:42:23,129 INFO    SystemMonitor:19880 [interfaces.py:start():190] Started gpu monitoring
2024-02-27 19:42:23,129 INFO    SystemMonitor:19880 [interfaces.py:start():190] Started memory monitoring
2024-02-27 19:42:23,130 INFO    SystemMonitor:19880 [interfaces.py:start():190] Started network monitoring
2024-02-27 19:42:23,135 ERROR   gpu       :19880 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-02-27 19:42:23,135 ERROR   gpu       :19880 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-02-27 19:42:23,135 ERROR   gpu       :19880 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-02-27 19:42:23,135 ERROR   gpu       :19880 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-02-27 19:42:23,155 DEBUG   HandlerThread:19880 [system_info.py:probe():151] Probing system
2024-02-27 19:42:23,157 DEBUG   HandlerThread:19880 [system_info.py:_probe_git():136] Probing git
2024-02-27 19:42:23,165 DEBUG   HandlerThread:19880 [system_info.py:_probe_git():144] Probing git done
2024-02-27 19:42:23,165 DEBUG   HandlerThread:19880 [system_info.py:probe():199] Probing system done
2024-02-27 19:42:23,165 DEBUG   HandlerThread:19880 [system_monitor.py:probe():223] {'os': 'Linux-5.4.0-150-generic-x86_64-with-glibc2.27', 'python': '3.11.5', 'heartbeatAt': '2024-02-27T14:12:23.155648', 'startedAt': '2024-02-27T14:12:22.378170', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/home/sadbhawna/Desktop/cs6910_assignment1/sweep_functionality.py', 'codePathLocal': 'sweep_functionality.py', 'codePath': 'sweep_functionality.py', 'git': {'remote': 'https://github.com/AbhishekKarmakar5/cs6910_assignment1.git', 'commit': '6bd33c7106a366eec851f1e823729a5cab5c29dc'}, 'email': 'YOUR_EMAIL', 'root': '/home/sadbhawna/Desktop/cs6910_assignment1', 'host': 'abhishek', 'username': 'sadbhawna', 'executable': '/home/sadbhawna/anaconda3/bin/python', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 1.7088749999999997, 'min': 1200.0, 'max': 3800.0}, 'cpu_freq_per_core': [{'current': 1.804, 'min': 1200.0, 'max': 3800.0}, {'current': 1.315, 'min': 1200.0, 'max': 3800.0}, {'current': 1.781, 'min': 1200.0, 'max': 3800.0}, {'current': 2.191, 'min': 1200.0, 'max': 3800.0}, {'current': 1.981, 'min': 1200.0, 'max': 3800.0}, {'current': 1.328, 'min': 1200.0, 'max': 3800.0}, {'current': 1.501, 'min': 1200.0, 'max': 3800.0}, {'current': 1.77, 'min': 1200.0, 'max': 3800.0}], 'disk': {'/': {'total': 883.450553894043, 'used': 125.09012603759766}}, 'gpu': 'NVIDIA GeForce GT 710', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GT 710', 'memory_total': 2090926080}], 'memory': {'total': 31.239574432373047}}
2024-02-27 19:42:23,165 INFO    HandlerThread:19880 [system_monitor.py:probe():224] Finished collecting system info
2024-02-27 19:42:23,165 INFO    HandlerThread:19880 [system_monitor.py:probe():227] Publishing system info
2024-02-27 19:42:23,166 DEBUG   HandlerThread:19880 [system_info.py:_save_conda():208] Saving list of conda packages installed into the current environment
2024-02-27 19:42:24,013 INFO    Thread-13 :19880 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240227_194222-73trvyvh/files/conda-environment.yaml
2024-02-27 19:42:25,123 INFO    MainThread:19880 [internal.py:handle_exit():76] Internal process exited
2024-02-27 19:42:25,133 INFO    cpu       :19880 [interfaces.py:monitor():140] Process cpu has exited.
2024-02-27 19:42:25,133 DEBUG   SystemMonitor:19880 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-02-27 19:42:25,134 DEBUG   SystemMonitor:19880 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-02-27 19:42:25,134 DEBUG   SystemMonitor:19880 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-02-27 19:42:25,135 INFO    memory    :19880 [interfaces.py:monitor():140] Process proc.memory.rssMB has exited.
2024-02-27 19:42:25,136 ERROR   gpu       :19880 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
